### ISSUE TITLE
Selecting jobs in Oban Web UI throws exception

### ISSUE THREAD

**Issue by nranas** [sorentwo/oban#1013]
Hi! After upgrading to oban_web 2.10.2 from 2.10.1 we are experiencing an issue in the Web UI.

### Environment

- oban 2.17.1
- oban_met 0.1.4
- oban_pro 1.2.2
- oban_web 2.10.2
- elixir 1.15.5
- erlang 26.0.2

### Current Behavior

When trying to select jobs by checking the checkboxes in the Oban.Web UI, the checkboxes are not checked. The following error is logged on the server:

```
UndefinedFunctionError
Oban.Web.DashboardLive.handle_event/3
function Oban.Web.DashboardLive.handle_event/3 is undefined or private
```
The event in this case is `pause-refresh`

```
UndefinedFunctionError: function Oban.Web.DashboardLive.handle_event/3 is undefined or private
  Module "Elixir.Oban.Web.DashboardLive", in Oban.Web.DashboardLive.handle_event/3
  File "lib/phoenix_live_view/channel.ex", line 487, in anonymous fn/3 in Phoenix.LiveView.Channel.view_handle_event/3
  File "/app/deps/telemetry/src/telemetry.erl", line 321, in :telemetry.span/3
  File "lib/phoenix_live_view/channel.ex", line 246, in Phoenix.LiveView.Channel.handle_info/2
  File "gen_server.erl", line 1077, in :gen_server.try_handle_info/3
  File "gen_server.erl", line 1165, in :gen_server.handle_msg/6
  File "proc_lib.erl", line 241, in :proc_lib.init_p_do_apply/3
```

### Expected Behavior

I expect that clicked checkboxes gets checked so that I can manually perform actions on jobs, e.g. retry, cancel, delete etc.

Please let me know if you need any more information!

**Comment by sorentwo** [comment]
Which version of `phoenix_live_view` and `phoenix_html` are you using? Also, which environment are you seeing this in?

**Comment by nranas** [comment]
- phoenix: 1.7.10
- phoenix_html: 3.3.3
- phoenix_live_view: 0.20.1

Operating System: Debian GNU/Linux 11 (bullseye)
Kernel: Linux 5.10.0-21-amd64
Architecture: x86-64

**Comment by sorentwo** [comment]
Sorry, I meant which mix environment; dev, prod, staging. I’m unable to reproduce that issue in any environment and wonder if it’s related to something in your dev environment.

**Comment by nranas** [comment]
Oh, sorry. We were seeing it both in dev and prod. Reverting back to `oban_web` 2.10.1 resolved the issue in both environments. I noticed now that the commit also updated the following deps in the mix.lock file:
- `castore` 1.0.4 -> 1.0.5
- `oban_met`  0.1.3 -> 0.1.4
- `phoenix_template 1.0.3 -> 1.0.4

I'll see if I can isolate the issue further.

**Comment by sorentwo** [comment]
Will you try upgrading `phoenix_html` to 4.0 and see if that fixes it? The biggest change in Web v2.10.2 was adding support for the latest `phoenix_html` release.

**Comment by nranas** [comment]
I redid the update from scratch now and this time `phoenix_live_view` got updated from 0.20.1 to 0.20.3 which wasn't in the last update and now it's working again so that must be the culprit. So it was just a bad update on our part, you can close this ticket. 
Sorry for wasting your time!

**Comment by nranas** [comment]
> Will you try upgrading `phoenix_html` to 4.0 and see if that fixes it? The biggest change in Web v2.10.2 was adding support for the latest `phoenix_html` release.

We need to stay on `phoenix_html` 3 for now since we use a library that does not yet support 4.0 but updating `phoenix_live_view` fixed it.

**Comment by sorentwo** [comment]
Ok, that's great to hear! Hopefully `phoenix_live_view` hits 1.0 soon and we have more stability.
